lib/forcegraph.js

```diff
@@ -682,6 +697,8 @@ define(["d3"], function (d3) {
       })
 
       nodes = intNodes.filter(function (d) { return d.o.node })
+      uplinkNodes = intNodes.filter(function (d) { return d.o.node && d.o.node.flags.uplink })
```

Comment:
This can be reduced to `nodes.filter(function (d) { return d.o.node.flags.uplink })`